benchmarks/src/CMakeLists.txt

```diff
@@ -1,26 +1,45 @@
-cmake_minimum_required(VERSION 3.10) # Set the minimum required version of CMake
+cmake_minimum_required(VERSION 3.10)
 
-project(SerializationBenchmark) # Name your project
+project(SerializatonTwitterBenchmark) # Name your project
 if (NOT CMAKE_BUILD_TYPE)
   message(STATUS "No build type selected, default to Release")
   set(CMAKE_BUILD_TYPE Release CACHE STRING "Choose the type of build." FORCE)
 endif()
+
 # Set C++ standard to C++17 or higher if needed
-set(CMAKE_CXX_STANDARD 23)
-set(CMAKE_CXX_STANDARD_REQUIRED ON)
+#set(CMAKE_CXX_STANDARD 23)
+#set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
 # Optionally, specify the C++ compiler directly (here we use clang++)
 set(CMAKE_C_COMPILER clang)
 set(CMAKE_CXX_COMPILER clang++)
 
+# set(CMAKE_OSX_ARCHITECTURES "arm64")
+
+# if(CMAKE_BUILD_TYPE STREQUAL "Debug")
```

Comment:
What I would suggest, instead of having commented out code that (presumably) one will uncomment manually, I would just add an option. Like so...

```cmake

option(SIMDJSON_SANITIZE_MEMORY "Sanitize memory" OFF)


if(SIMDJSON_SANITIZE_MEMORY)
  message(STATUS "Setting the memory sanitizer.")
  add_compile_options(
      -fsanitize=memory -fno-sanitize-recover=all
  )
  link_libraries(
      -fsanitize=memory -fno-sanitize-recover=all
  )
endif()
```

And then...

```cmake
if(NOT CMAKE_BUILD_TYPE)
  if(SIMDJSON_SANITIZE OR SIMDJSON_SANITIZE_UNDEFINED)
    message(STATUS "No build type selected and you have enabled the sanitizer, \
default to Debug. Consider setting CMAKE_BUILD_TYPE.")
    message(STATUS "Setting debug optimization flag to -O1 to help sanitizer.")
    set(CMAKE_CXX_FLAGS_DEBUG "-O1" CACHE STRING "" FORCE)
    set(CMAKE_BUILD_TYPE Debug CACHE STRING "Choose the type of build." FORCE)
  else()
    message(STATUS "No build type selected, default to Release")
    set(CMAKE_BUILD_TYPE Release CACHE STRING "Choose the type of build." FORCE)
  endif()
endif()
```